bitcoinsuite-chronik-client/proto/chronik.proto

```diff
@@ -293,14 +422,16 @@ message WsSub {
         WsSubBlocks blocks = 2;
         // Subscription to a script
         WsSubScript script = 3;
+        // Subscription to a token ID
+        WsSubTokenId token_id = 4;
     }
 }
 
 // Subscription to blocks. They will be sent any time a block got connected,
 // disconnected or finalized.
 message WsSubBlocks {}
 
-// Subscription to a script. They will be send every time a tx spending the
+// Subscription to a script. They will be sent every time a tx spending the
```

Comment:
1. The `WsSubScript` is not sent "every time a tx spending..." right? It's only sent on subscription
2. The keys here in chronik-client proto for in-node are

```
export interface WsSubScript {
    /** Script type to subscribe to ("p2pkh", "p2sh", "p2pk", "other"). */
    scriptType: string;
    /**
     * Payload for the given script type:
     * - 20-byte hash for "p2pkh" and "p2sh"
     * - 33-byte or 65-byte pubkey for "p2pk"
     * - Serialized script for "other"
     */
    payload: Uint8Array;
}
```

`chronik/chronik-proto/proto/chronik/proto` in the monorepo shows

```
message WsSubScript {
    // Script type to subscribe to ("p2pkh", "p2sh", "p2pk", "other").
    string script_type = 1;
    // Payload for the given script type:
    // - 20-byte hash for "p2pkh" and "p2sh"
    // - 33-byte or 65-byte pubkey for "p2pk"
    // - Serialized script for "other"
    bytes payload = 2;
}
```

is script_type not needed here?

Reply:
I just copy-pasted this from the ABC codebase, should be fixed upstream first IMO